libs/ledger-state/app/Main.hs

```diff
@@ -14,7 +14,10 @@ import System.IO
 data Opts = Opts
   { -- | Path to the CBOR encoded NewEpochState data type, which will be used to
     -- load into sqlite database
-    optsLedgerStateBinaryFile :: Maybe FilePath,
+    optsNewEpochStateBinaryFile :: Maybe FilePath,
+    -- | Path to the CBOR encoded NewEpochState data type, which will be used to
+    -- load into sqlite database
```

Comment:
Yep, copy and paste error. Fixed and added a comment about how these options are used:

A bit more insight into options, at least into how they are currently used in this PR. This will  change after new serialization in this PR is realeased:

* `optsNewEpochStateBinaryFile` is for reading a previously serialized `NewEpochState` produced by cadano-cli` and is used to populate sqlite database
* `optsEpochStateBinaryFile` is used for grabbing data from sqlite, constructing `EpochState` (in a new format) and writing it into the cbor serialized file